meetings/2024-04-24.md

```diff
@@ -28,41 +35,101 @@
+* eP: we will cotniue on the chat. 
+
+ACTION: eP: to share a link with a diff between WD and ED
+
+
+### WIP Implementation Feedback
+* VB: We'll allocate some time for implementation feedback or interest to implement. Links to products/projects and demos welcome. 
+
+#### Tentative, preliminary, pragmatic plans for a public CSS instance
+* VB: proposed by ME
+URL: https://teamid.live/
+URL: https://meisdata.io/
+URL: https://opencollective.com/meisdata/projects/teamid
+URL: https://hackmd.io/PTHgp2BTSM2XyVNi8omI_g?view
+brainstorming, things to consider..., monetization
+
+* ME: will move the plans in his head back and forth (had mike problems)
 
 ### WG Charter
 
 #### Please upvote and downvote proposed WG names 
 URL: https://github.com/solid/solid-wg-charter/issues/75
-URL: https://github.com/solid/solid-wg-charter/issues/73
 
+* VB: Discussion at https://github.com/solid/solid-wg-charter/issues/73
 * VB: We agreed to decide on final name today.
+* SC: I've made minor updates to PAC's code ( https://champin.net/2024/solid-name.html ) : https://csarven.ca/tmp/wg-name-poll.html , e.g., shows voters. Feel free to update yours with this. I don't plan to keep my version on my site long. We could consider moving it over to Solid ecosystem monitor ( ping @Virginia Balseiro ) to be used for any issue / poll / reactions stuff. 
+* SC: One thing to mind / pass on to PAC (Pierre-Antoine Champin) / W3C Team here is that majority of the the names and voters are from the Solid community. The names may be leaning with Solid-perspective, but not necessarily ideal for the WG in and itself that's intended to at least not come across as Solid-centric. There is no perfect name and it'll mean different things to everyone. For example, "Linked Web Storage WG" may be quite meaningful to SW/LD/Solid folks, but perhaps not great for anyone outside that view - some may even dislike the whole "Linked" Stuff. Similarly "Decentralised Online Storage" could be well-rounded for a lot of people but "decentralised" can also mean different things to everyone. And, for some continuity with "Solid", I can see that being reused with a new acronym - but that might also add more confusion. Just wanted to raise these types of considerations towards the final naming that the wider W3C community can consider.
+* PAC: I agree with above, Linked Web Storage is the most upvoted and least controvesial names. I share some concerns about Linked. Nothing in the charter mentiones linked data, currently it is not core to the charter. Some people may have reactions. That being said I don't want to impose a single view. The github issue is a good reference for the future.
+* PAC: I sent the current version of the charter, didn't get any feedback yet. I got a confirmation from the 3rd chair, Eric Prud'hommeaux who has more exprience prevously working at W3C.
+* eP: Glad to hear Eric will be involved especially because we want to rely heavily on shapes. 
```

Comment:
```suggestion
* SC: I've made minor updates to PAC's code (<https://champin.net/2024/solid-name.html>): <https://csarven.ca/tmp/wg-name-poll.html>, e.g., shows voters. Feel free to update yours with this. I don't plan to keep my version on my site long. We could consider moving it over to Solid ecosystem monitor (ping @VirginiaBalseiro) to be used for any issue / poll / reactions stuff. 
* SC: One thing to mind / pass on to PAC (Pierre-Antoine Champin) / W3C Team here is that majority of the the names and voters are from the Solid community. The names may be leaning with Solid-perspective, but not necessarily ideal for the WG in and of itself, that's intended to at least not come across as Solid-centric. There is no perfect name, and it'll mean different things to everyone. For example, "Linked Web Storage WG" may be quite meaningful to SW/LD/Solid folks, but perhaps not great for anyone outside that view — some may even dislike the whole "Linked" Stuff. Similarly "Decentralised Online Storage" could be well-rounded for a lot of people but "decentralised" can also mean different things to everyone. And, for some continuity with "Solid", I can see that being reused with a new acronym — but that might also add more confusion. Just wanted to raise these types of considerations towards the final naming that the wider W3C community can consider.
* PAC: I agree with above, _Linked Web Storage_ is the most upvoted and least controversial name. I share some concerns about _Linked_. Nothing in the charter mentions linked data; currently it is not core to the charter. Some people may have reactions. That being said I don't want to impose a single view. The github issue is a good reference for the future.
* PAC: I sent the current version of the charter, didn't get any feedback yet. I got a confirmation from the 3rd chair, Eric Prud'hommeaux who has more exprience previously working at W3C.
* eP: Glad to hear Eric will be involved especially because we want to rely heavily on shapes. 
```